src/platform/Kconfig

```diff
@@ -530,6 +530,13 @@ config HAVE_AGENT
 	  with DMA based scheduling, where asynchronous interrupts
 	  can potentially starve the agent.
 
+config AMS
+	bool "Enable Async Messaging Service"
+	default n
```

Comment:
How about this: please change this to `y`. So it's compiled by the CI. After it's reviewed and merged, submit a new PR with just one change - from `y` to `n`

Reply:
Or define it in e.g. ace15 board file, so it gets build by default?

Reply:
> How about this: please change this to y. So it's compiled by the CI. After it's reviewed and merged, submit a new PR with just one change - from y to n

There is a way to do this without going back in forth in the permanent git history: submit another draft, do-not-review PR with the same code as here + 1 throw away commit that turns it on.